### DISCUSSION TITLE
Need a convenient API for adding issue.

### DISCUSSION THREAD

**Discussion by cyal1** [PortSwigger/burp-extensions-montoya-api#69] [Q&A]
Is there a method similar to Burp Suite's Wiener API  `callbacks.addScanIssue` in montoya? It should be easy and convenient to add the issue in Wiener API, I find and report some issues in the context menu program without creating a Scanner check.

**Comment by Hannah-PortSwigger** [comment]
`api.sitemap().add(AuditIssue)` should be what you are looking for 🙂

**Reply by cyal1** [reply]
Thanks
